meilix-metapackages-2_1.0_all/control

```diff
@@ -0,0 +1,26 @@
+### Commented entries have reasonable defaults.
+### Uncomment to edit them.
+Source: meilix-metapackages-2
+Section: misc
+Priority: extra
+# Homepage: <enter URL here; no default>
+Standards-Version: 3.9.2
+
+Package: meilix-metapackages-2
+# Version: <enter version here; defaults to 1.0>
```

Comment:
Version needs to be in commented because all the different packages are with the different version and I have done the same in the last metapackage also.